lightning: add using lighting to migrate aurora data tutorial (#4857)
pingcap/docs: migrate-from-aurora-using-lightning.md

    ```
    [mydumper]
    # Data source directory
    data-source-dir = "s3://bucket-name/data-path"
    ```

2. Configure the target TiDB cluster as follows:

    ```
    [tidb]
    # The target cluster information. Fill in one address of tidb-server.
    host = "172.16.31.1"
    port = 4000
    user = "root"
    password = ""
    # The PD address of the cluster.
    pd-addr = "127.0.0.1:2379"
    ```

3. Configure the backend mode:

    ```
    [tikv-importer]
    # Uses Local-backend.
    backend = "local"
    # The storage path of local temporary files. Ensure that the corresponding directory does not exist or is empty and that the disk capacity is large enough for storage.
    sorted-kv-dir = "/path/to/local-temp-dir"
    ```

4. Configure the file routing.

    ```
    [mydumper]
    no-schema = true

    [mydumper.files]
    # Uses single quoted strings to avoid escaping.
    pattern = '(?i)^(?:[^/]*/)*([a-z0-9_]+)\.([a-z0-9_]+)/(?:[^/]*/)*(?:[a-z0-9\-_.]+\.(parquet))$'
    schema = '$1'
    table = '$2'
    type = '$3'
    ```

> **Note:**
>

> > - The example in this document uses the Local-backend for best performance. You can also choose TiDB-backend or Importer-backend according to your need. For detailed introduction of the three backend modes, see [TiDB Lightning Backends](/tidb-lightning/tidb-lightning-backends.md).

Reviewer: ```suggestion
> - The above example uses the Local-backend for best performance. You can also choose TiDB-backend or Importer-backend according to your need. For detailed introduction of the three backend modes, see [TiDB Lightning Backends](/tidb-lightning/tidb-lightning-backends.md).
```

> - Because the path for exporting snapshot data from Aurora is different from the default file naming format supported by TiDB Lightning, you need to set additional file routing configuration.
> - If TLS is enabled in the target TiDB cluster, you also need to configure TLS.

> For other configuration, see [TiDB Lightning Configuration](/tidb-lightning/tidb-lightning-configuration.md).

Reviewer: ```suggestion
For other configurations, see [TiDB Lightning Configuration](/tidb-lightning/tidb-lightning-configuration.md).
```

## Step 4: Create table schema

Because the snapshot data exported from Aurora to S3 does not contain the SQL statement file used to create database tables, you need to manually export and import the table creation statements corresponding to the database tables into TiDB. You can use Dumpling + TiDB Lightning to create all table schemas:

1. Use Dumpling to export table schema files:

    ```
    ./dumpling --host 127.0.0.1 --port 4000 --user root --password password --no-data --output ./schema --filter "mydb.*"
    ```

    > **Note**
    >
    > - Set the parameters of the data source address and the path of output files according to your actual situation.
    > - If you need export all database tables, you do not need to set the `--filter` parameter. If you only need to export some of the database tables, configure `--filter` according to [table-filter](https://github.com/pingcap/tidb-tools/blob/master/pkg/table-filter/README.md).

2. Use TiDB Lightning to create table schemas:

    ```
    ./tidb-lightning -config tidb-lightning.toml -d ./schema -no-schema=false
    ```

>     In this example, TiDB Lightning is only used to create table schemas, so you need to execute the above command quickly. At a regular speed, ten table creation statements can be executed in one second.

Reviewer 2: here we just want to express that "this command can be executed very quickly", the original Chinese description is not accurate, so sorry for that.

>     In this example, TiDB Lightning is only used to create table schemas, so you need to execute the above command quickly. At a regular speed, ten table creation statements can be executed in one second.

Author: According to @glorv, "应迅速执行以上命令" means that "TiDB Lightning can quickly process the command".
```suggestion
    In this example, TiDB Lightning is only used to create table schemas and can quickly process the above command. At a regular speed, ten table creation statements can be executed in one second.
```
